deploy/003_deploy_optimism_spokepool.ts

```diff
@@ -1,26 +1,33 @@
 import "hardhat-deploy";
-import { HardhatRuntimeEnvironment } from "hardhat/types/runtime";
+import hre from "hardhat";
+import { getContractFactory } from "../test/utils";
```

Comment:
Given that we're going to rely on this for prod, any thoughts on moving it from `src/test/utils` => `src/utils` ?

Reply:
Great call these all seem like useful functions to have in `src`